viewer/packages/pointclouds/src/PotreeGroupWrapper.ts

```diff
@@ -90,7 +90,6 @@ export class PotreeGroupWrapper extends THREE.Object3D {
   }
 
   addPointCloud(node: PotreeNodeWrapper): void {
-    this.potreeGroup.add(node.octree);
```

Comment:
This is removed here because the octree is reparented right after in the `PointCloudNode` constructor.